README.md

```diff
@@ -11,13 +11,18 @@ An attempt at collecting some of the collections of accessible components but al
 - [A11y Project Patterns](http://a11yproject.com/patterns.html) - An accessible widget & pattern library
 - [Reakit](https://reakit.io/) - Accessible, Composable and Customizable components for React
 
+## Contrast checker
+
+- [a11y-contrast](https://github.com/darekkay/a11y-contrast) — A CLI utility to calculate/verify accessible magic numbers for a color palette.
+
 ## Autocomplete
 
 - [Downshift](https://github.com/paypal/downshift) - React autocomplete/dropdown/select/combobox components by [Paypal](https://github.com/paypal)
 - [React Autosuggest](https://github.com/moroshko/react-autosuggest) - WAI-ARIA compliant React autosuggest component More info at [react-autosuggest.js.org](http://react-autosuggest.js.org)
 
 ## Modal
 
+- [a11y-dialog](https://github.com/KittyGiraudel/a11y-dialog) - A very lightweight and flexible accessible modal dialog script.
```

Comment:
This seems like a duplication of the resource below with the same name. So I'm thinking one of them needs to go and one should be kept. Will look into it.

Reply:
Looked into this and the link below actually redirects to the link you added. So I think the proper thing to do is to just update the link.

Reply:
I see. Now that I'm looking at this closer, it looks like this readme is taking the liberty of using all-caps names for all components. I'll follow that convention for now. I think this is an improvement to replace the link since we're now giving the correct author credit.